INSTALL.md

```diff
@@ -105,6 +105,81 @@ For Example:
 ./tools/tpm2_getrandom 4
 ```
 
+### Build example
+
+#### Fedora 27
+
+Here we are going to build tpm2-tools with:
+* tpm2-tss: <https://github.com/tpm2-software/tpm2-tss>
+* tpm2-abrmd: <https://github.com/tpm2-software/tpm2-abrmd>
+* TPM simulator: <https://downloads.sourceforge.net/project/ibmswtpm2/ibmtpm974.tar.gz>
+
+First we need to satisfy the dependencies for each item named above except for the simulator, in the next code block we install a mix of packages that should get the work done.
+
+
+```
+$ sudo dnf -y update && sudo dnf -y install automake libtool \
+autoconf autoconf-archive libstdc++-devel gcc pkg-config \
+uriparser-devel libgcrypt-devel dbus-devel glib2-devel \
+compat-openssl10-devel libcurl-devel PyYAML
+
+```
+For more detailed information about the dependencies of tpm2-tss and tmp2-abrmd, please consult the links provided above.
+
+Now we can start building the packages  
+
+* ##### Tpm2-tss
+```
+$ git clone git://github.com/tpm2-software/tpm2-tss.git
+$ cd tpm2-tss
+$ ./bootstrap
+$ ./configure --prefix=/usr
+$ make -j5
+```
+
+* ##### Tpm2-abrmd
+```
+$ git://github.com/tpm2-software/tpm2-abrmd.git
+$ cd tpm2-abrmd
+$ ./bootstrap
+$ ./configure --with-dbuspolicydir=/etc/dbus-1/system.d
+--with-udevrulesdir=/usr/lib/udev/rules.d
+--with-systemdsystemunitdir=/usr/lib/systemd/system
+--libdir=/usr/lib64 --prefix=/usr
+$ make -j5
```

Comment:
same here, please add a `sudo make install` command.

Reply:
Please no, here -- this practice is suitable for a developmental machine, perhaps, but not for much more.  

Clearly a 'spec' file for Fedora exists -- use it and make packages

Reply:
@herrold of course I know that there are packages for tpm2-{tss,abrmd,tools} available in Fedora, I'm the one who maintains them...

Point is what this PR suggests to add information on how to **build from source**.  For whatever reasons (testing the latest master, development purposes, etc) people may want to build from source and if there are already instructions for that, then I think that we should include information on how what was built can be installed.